packages/components/src/components/Flag/index.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import React from 'react';
+import { FlagType } from '../../support/types';
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+`;
+
+interface Props {
+    className?: string;
+    country: FlagType;
+    size?: number;
+}
+
+const Flag = ({ size = 24, country, className }: Props) => {
+    return (
+        <Wrapper>
+            <img
+                // eslint-disable-next-line global-require, import/no-dynamic-require
+                src={require(`../../images/flags/${country.toLowerCase()}.svg`)}
```

Comment:
Same comment as on the other PR. SVG's can be accessed directly (like it's done in other parts of the project) rather than being dynamically imported.

Reply:
The problem is if the SVG is accessed directly, it is embedded in the html, resulting in huge code, which is slow in the browser. This way it is fast. Also I think that using require for images is a common practice, see example code in https://github.com/cyrilwanner/next-optimized-images